src/main/java/com/github/javafaker/Finance.java

```diff
@@ -18,10 +18,10 @@ protected Finance(Faker faker) {
 
     private static final Map<String, String> countryCodeToBasicBankAccountNumberPattern =
             createCountryCodeToBasicBankAccountNumberPatternMap();
-//
-    public String creditCard() {
-        CreditCardType type = randomCreditCardType();
-        final String key = String.format("credit_card.%s", type.toString().toLowerCase());
+
+    public String creditCard(CreditCardType creditCardType)
+    {
```

Comment:
can you keep the brace formatting the same please?

Reply:
Yup, fixed that. Thanks.